CHANGELOG.rst

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
 * Fixes input and textarea fields in not aligned inline groups
 * Fixes select field width in table
+* Adding missing labels to datetime
```

Comment:
i think should be more specific 
`Added missing labels to datetime fields that are already part of the field box`